sdk/python/kfp_tekton/tekton.py

```diff
@@ -31,34 +32,64 @@ class AnySequencer(ContainerOp):
                 because the pipeline will generate a unique new name in case of conflicts.
 
         any: List of `Conditional` containerOps that deploy together with the `main`
-                containerOp.
+                containerOp, or the condtion that must meet to continue.
     """
     def __init__(self,
-                 any: List[dsl.ContainerOp],
+                 any: Iterable[Union[dsl.ContainerOp, ConditionOperator]],
                  name: str = None,):
 
         tasks_list = []
+        condition_list = []
         for cop in any:
-            cop_name = sanitize_k8s_name(cop.name)
-            tasks_list.append(cop_name)
+            if isinstance(cop, dsl.ContainerOp):
+                cop_name = sanitize_k8s_name(cop.name)
+                tasks_list.append(cop_name)
+            elif isinstance(cop, ConditionOperator):
+                condition_list.append(cop)
+
         task_list_str = ",".join(tasks_list)
+        arguments = [
+                    "--namespace",
+                    "$(context.pipelineRun.namespace)",
+                    "--prName",
+                    "$(context.pipelineRun.name)",
+                    "--taskList",
+                    task_list_str
+                ]
+        conditonArgs = processConditionArgs(condition_list)
+        arguments.extend(conditonArgs)
 
         super().__init__(
             name=name,
             image="dspipelines/any-sequencer:latest",
             command="any-taskrun",
-            arguments=[
-                        "-namespace",
-                        "$(context.pipelineRun.namespace)",
-                        "-prName",
-                        "$(context.pipelineRun.name)",
-                        "-taskList",
-                        task_list_str
-                    ],
+            arguments=arguments,
         )
 
 
-def after_any(any: List[dsl.ContainerOp], name: str = None):
+def processOperand(operand) -> str:
+    if isinstance(operand, dsl.PipelineParam):
+        return "results_" + operand.op_name + "_" + operand.name
+    else:
+        return operand
+
+
+def processCondition(condition: ConditionOperator) -> str:
+    conditionStr = processOperand(condition.operand1)
+    conditionStr = conditionStr + " " + condition.operator + " "
+    conditionStr = conditionStr + processOperand(condition.operand2)
```

Comment:
Looks a bit confusing. How about the following?

```python
op1 = processOperand(condition.operand1)
op2 = processOperand(condition.operand2)
conditionStr = f"{op1} {condition.operator} {op2}"
```

Reply:
Sure, will do.